HACKING.md

```diff
@@ -0,0 +1,118 @@
+## Introduction
+LU is build with the [Entity–component–system](https://en.wikipedia.org/wiki/Entity_component_system) which you may know from the Unity Engine. In short that means everything that you see on screen (and more) is a GameObject on which Components can be attached. These components give the object functionality like rendering, physics and so on. 
+
+LU used two different types of servers to communicate. These are: 
+* The authentication server (Uchu.Auth), which handles login and redirection to the world server. 
+* The world server, which handles two things
+  * Character selection (Uchu.Char)
+  * Main game (Uchu.World)
+
+**Note:** A world server in Uchu only handles one world but there can be multiple world servers in one Uchu instance. They can even serve the same map if one world server has reached the maximum amount of player. 
```

Comment:
There's a few small typos here and there, replace 'player' with 'players'

Reply:
Interesting typo. Imagine the server being like: "Nah that's too much player for today gonna make a new world server"